pkg/ddevapp/global_dotddev_assets/commands/web/craft

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#ddev-generated
+## Description: Run Craft CMS command inside the web container
+## Usage: craft [flags] [args]
+## Example: "ddev craft db/backup" or "craft db/backup ./my-backups" (see https://craftcms.com/docs/4.x/console-commands.html)
+## ProjectTypes: php
+## ExecRaw: true
+
+php ./craft "$@"
```

Comment:
Probably don't need the `php` here but it does no harm.

Reply:
Right, I just followed the convention set by the other commands, for example https://github.com/drud/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/web/artisan